src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3795,6 +3806,15 @@ public sealed partial class WeakReference<T> : System.Runtime.Serialization.ISer
 }
 namespace System.Buffers
 {
+    public abstract partial class ArrayPool<T>
```

Comment:
I couldn't find deleted lines corresponding to the APIs for ArrayPool.

Reply:
I could not delete `src/libraries/System.Buffers/ref/System.Buffers.cs` because of it is part of the netstandard hack that Eric commented on above as well.